src/net.h

```diff
@@ -656,6 +656,84 @@ class V2Transport final : public Transport
     Info GetInfo() const noexcept override EXCLUSIVE_LOCKS_REQUIRED(!m_recv_mutex);
 };
 
+/**
+ * Network traffic (bytes and number of messages). Split by direction, network, connection type and message type.
+ */
+class NetStats
```

Comment:
Would there be any benefit to moving this class outside of `net.{h,cpp}`? It's quite a lot of code.

(Though, much of its verbosity is the formatting style used that often reads top-to-bottom rather than left-to-right. I prefer the latter.)

Reply:
No opinion.